src/processScript/transform.ts

```diff
@@ -780,7 +780,7 @@ export function transform(
 		)
 	}
 
-	function processFakeSubscriptObject(fakeSubscriptObjectName: string) {
+	function processFakeSubscriptObject(fakeSubscriptObjectName: string, secLevel: number) {
```

Comment:
please can `secLevel` be renamed to `seclevel`?
The word "seclevel" has become a real word in the context of hackmud whereas "`secLevel`" would be an abbreviation of "security level" so it should really be named `securityLevel`. But I'd rather the name `seclevel` than `securityLevel`.